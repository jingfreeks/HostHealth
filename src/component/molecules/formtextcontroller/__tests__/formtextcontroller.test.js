import React from 'react';
import {render} from '@testing-library/react-native';
import Formtextcontroller from '../formtextcontroller';

jest.mock('react-hook-form', () => {
  return {
    ...jest.requireActual('react-hook-form'),
    Controller: () => jest.fn(),
    useFormContext: () => ({
      // eslint-disable-next-line @typescript-eslint/no-empty-function
      control: () => jest.fn(),
    }),
  };
});
describe('Form Text Controller Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<Formtextcontroller />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
