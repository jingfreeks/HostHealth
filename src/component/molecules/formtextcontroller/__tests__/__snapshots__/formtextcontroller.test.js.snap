// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Text Controller Component Should work as expected to get snapshot 1`] = `null`;
